examples/burst_over_time_on_command.rs

```diff
@@ -0,0 +1,178 @@
+use bevy::{
+    log::LogPlugin,
+    math::Vec3Swizzles,
+    prelude::*,
+    render::{
+        camera::{Projection, ScalingMode},
+        render_resource::WgpuFeatures,
+        settings::WgpuSettings,
+        RenderPlugin,
+    },
+};
+use bevy_inspector_egui::quick::WorldInspectorPlugin;
+
+use bevy_hanabi::prelude::*;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut wgpu_settings = WgpuSettings::default();
+    wgpu_settings
+        .features
+        .set(WgpuFeatures::VERTEX_WRITABLE_STORAGE, true);
+
+    App::default()
+        .insert_resource(ClearColor(Color::DARK_GRAY))
+        .add_plugins(
+            DefaultPlugins
+                .set(LogPlugin {
+                    level: bevy::log::Level::WARN,
+                    filter: "bevy_hanabi=warn,spawn=trace".to_string(),
+                })
+                .set(RenderPlugin { wgpu_settings }),
+        )
+        .add_system(bevy::window::close_on_esc)
+        .add_plugin(HanabiPlugin)
+        .add_plugin(WorldInspectorPlugin::default())
+        .add_startup_system(setup)
+        .add_system(update)
+        .run();
+
+    Ok(())
+}
+
+#[derive(Component)]
+struct Ball {
+    velocity: Vec2,
+}
+
+const BOX_SIZE: f32 = 2.0;
+const BALL_RADIUS: f32 = 0.05;
+
+fn setup(
+    mut commands: Commands,
+    mut effects: ResMut<Assets<EffectAsset>>,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    let mut camera = Camera3dBundle::default();
+    let mut projection = OrthographicProjection::default();
+    projection.scaling_mode = ScalingMode::FixedVertical(2.);
+    projection.scale = 1.2;
+    camera.transform.translation.z = projection.far / 2.0;
+    camera.projection = Projection::Orthographic(projection);
+    commands.spawn(camera);
+
+    commands
+        .spawn(PbrBundle {
+            mesh: meshes.add(Mesh::from(shape::Quad {
+                size: Vec2::splat(BOX_SIZE),
+                ..Default::default()
+            })),
+            material: materials.add(StandardMaterial {
+                base_color: Color::BLACK,
+                unlit: true,
+                ..Default::default()
+            }),
+            ..Default::default()
+        })
+        .insert(Name::new("box"));
+
+    let ball = commands
+        .spawn(PbrBundle {
+            mesh: meshes.add(Mesh::from(shape::UVSphere {
+                sectors: 32,
+                stacks: 2,
+                radius: BALL_RADIUS,
+            })),
+            material: materials.add(StandardMaterial {
+                base_color: Color::WHITE,
+                unlit: true,
+                ..Default::default()
+            }),
+            ..Default::default()
+        })
+        .insert(Ball {
+            velocity: Vec2::new(1.0, 2f32.sqrt()),
+        })
+        .insert(Name::new("ball"))
+        .id();
+
+    let spawner = Spawner::new(32.0.into(), 0.5.into(), std::f32::INFINITY.into()).with_time(0.5);
```

Comment:
nit: I'd set an initial time > 0.5 to avoid confusion, and maybe add a comment:
```suggestion
    // Set an initial time of 1.0s greater than the burst duration 0.5s to prevent spawning on start.
    let spawner = Spawner::new(32.0.into(), 0.5.into(), std::f32::INFINITY.into()).with_time(1.0);
```